File: Java/src/secao07/Exercicio06.java
package secao07;

import java.util.Scanner;

public class Exercicio06 {
    public static void main(String[] args) {
        // var
        int num, multiplicacao;
        Scanner teclado = new Scanner(System.in);

        // input
        System.out.println("Informe um número de 1 a 10: ");
        num = teclado.nextInt();

        // processing
        while (num > 10) {
            System.out.println("O número informeado não pode ser maior que 10.");
            System.out.println("Informe um número de 1 a 10: ");
            num = teclado.nextInt();
        }
        System.out.println("===================");
        System.out.println("Tabuada do " + num);
        for (int i = 0; i <= 10; i++) {
            multiplicacao = num * i;
            System.out.println(num + " x " + i + " = " + multiplicacao);
        }
        teclado.close();
    }
}

File: Python/secao04/Exercicio06.py
# input
numero = int(input("Informe um número: "))

# processing
while numero > 10:
    print("O número deve ser maior que 10.") 
    numero = int(input("Informe novamente um número: "))

print("Tabuada do {0}:".format(numero))

for i in range(0, 11):
    valor = numero * i
    print("{0} x {1} = {2}".format(numero, i, valor))